__tests__/getArticles.test.js

```diff
@@ -0,0 +1,63 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const data = require("../db/data/test-data");
+const seed = require("../db/seeds/seed");
+
+beforeEach(() => {
+  return seed(data);
+});
+afterAll(() => {
+  return db.end();
+});
+
+describe("GET /api/topics", () => {
```

Comment:
I think you may have written the wrong end point in this describe here

Reply:
Corrected